fineract-provider/build.gradle

```diff
@@ -55,7 +55,8 @@ buildscript {
         classpath "gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:2.2.2"
         classpath "net.ltgt.gradle:gradle-errorprone-plugin:1.2.1"
         classpath "com.diffplug.spotless:spotless-plugin-gradle:5.1.0"
-        classpath "io.swagger.core.v3:swagger-gradle-plugin:2.1.4"
+        classpath "io.swagger.core.v3:swagger-gradle-plugin:2.1.3"
```

Comment:
Are you sure about this change? Does the newer version bring some issues?

Reply:
Hi @thesmallstar Yes, you're right, this change was not necessary. I have created a new PR for this. Please review #1206